core/src/create.jl

```diff
@@ -22,11 +22,14 @@ end
 
 function LinearResistance(db::DB, config::Config)::LinearResistance
     static = load_structvector(db, config, LinearResistanceStaticV1)
-    return LinearResistance(static.node_id, static.resistance)
+    active = coalesce.(static.active, true)
```

Comment:
Do I understand correctly that the 3 valid values for "active" are, true, false, and missing? Would it be easier to disallow missing and default to true?

Reply:
Yes, that can be handled in the parse_static function of https://github.com/Deltares/Ribasim/pull/424.